soccer/writers.py

```diff
@@ -223,10 +225,11 @@ class Json(BaseWriter):
 
     def generate_output(self, result):
         if not self.output_filename:
-            click.echo(json.dumps(result, indent=4, separators=(',', ': ')))
+            click.echo(json.dumps(result, indent=4, separators=(',', ': '), ensure_ascii=False))
         else:
-            with open(self.output_filename, 'w') as json_file:
-                json.dump(result, json_file, indent=4, separators=(',', ': '))
+            with io.open(self.output_filename, 'w', encoding='utf-8') as json_file:
```

Comment:
whats special abt the io module? i ask because in the line below, you passed a parameter ensure_ascii=False...wont that take care of it?